### PR TITLE
Add upkeep bullets for 2022

### PR DESCRIPTION
**Pull request by tracykteal**
Updating tidy_upkeep checklist for 2022

### CODE REVIEW

File: R/tidy-upkeep.R [hunk 1]
```diff
@@ -85,6 +85,20 @@ upkeep_checklist <- function(year = NULL) {
       ""
     )
   }
+if (year <= 2022) {
+    bullets <- c(bullets,
+     "2022",
+     "",
+     todo("`usethis::use_tidy_coc()`"),
+     todo("Update errors to rlang 1.0.0"),
```
Comment (jennybc) [inline R/tidy-upkeep.R:93]: Is there a guide for this?

File: R/tidy-upkeep.R [hunk 2]
```diff
@@ -85,6 +85,20 @@ upkeep_checklist <- function(year = NULL) {
...
+     todo("Update pkgdown site using instructions at <https://tidytemplate.tidyverse.org>"),
+     todo("Re-publish released site using [r-lib/pkgdown#2051](https://github.com/r-lib/pkgdown/pull/2051)"),
```
Comment (jennybc) [inline R/tidy-upkeep.R:95]: Obviously we'll change this once the PR is merged

File: R/tidy-upkeep.R [hunk 3]
```diff
@@ -85,6 +85,20 @@ upkeep_checklist <- function(year = NULL) {
...
+     todo("Ensure pkgdown `development` is `mode: auto` in pkgdown config"),
```
Comment (jennybc) [inline R/tidy-upkeep.R:96]: FWIW this is included in the proposed pkgdown PR re: updating a released site.

File: R/tidy-upkeep.R [hunk 4]
```diff
@@ -85,6 +85,20 @@ upkeep_checklist <- function(year = NULL) {
...
+     todo("Handle and close any still-open `master` --> `main` issues"),
+     todo("Update README badges"),
```
Comment (jennybc) [inline R/tidy-upkeep.R:98]: We don't really say what this means or how to do it. But it's also very fiddly for little pay-off, so I'm not sure what to do about this.

**Comment by jennybc** [comment]
I've noted some loose ends but will merge anyway, to get the ball rolling.

File: R/tidy-upkeep.R (hunk 1 above)
Comment (tracykteal) [reply R/tidy-upkeep.R:93]: I think @lionel- has this in progress, so we'll update once it's available

File: R/tidy-upkeep.R (hunk 3 above)
Comment (tracykteal) [reply R/tidy-upkeep.R:96]: should we take it out of the list then?

File: R/tidy-upkeep.R (hunk 1 above)
Comment (lionel-) [reply R/tidy-upkeep.R:93]: There's https://rlang.r-lib.org/reference/topic-error-call.html and a guide about error chaining coming up.

Comment (lionel-) [reply R/tidy-upkeep.R:93]: Here is the topic about error chaining: https://rlang.r-lib.org/reference/topic-error-chaining.html
And the one about cli formatting: https://rlang.r-lib.org/reference/topic-condition-formatting.html

Comment (jennybc) [reply R/tidy-upkeep.R:93]: OK added.

**Pull request merged**